pkg/ui/src/views/app/components/layoutSidebar/index.tsx

```diff
@@ -49,10 +71,10 @@ export default class extends React.Component<{}, {}> {
   render() {
     return <nav className="navigation-bar">
       <ul className="navigation-bar__list">
-        <IconLink to="/overview" icon={homeIcon} title="Overview" />
+        <IconLink to="/overview" icon={homeIcon} title="Overview" indexFor={["/nodes", "/node"]} />
```

Comment:
A comment on `IconLinkProps` explaining the semantics of `indexFor` would be nice — I assume that it does a prefix match, so e.g. `/nodes/5` will cause the `/overview` link to be highlighted? Unclear that that would work from [this SO answer about `router.isActive`](https://stackoverflow.com/questions/43750056/checking-router-isactive-on-nested-routes-in-react?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa)

Reply:
yes, I spent a little while trying to figure out an obvious name for the prop... just changed it to `activeFor`, do you think that's clear enough, particularly after unifying all active class logic?